client/my-sites/sidebar-unified/use-site-menu-items.js

```diff
@@ -54,6 +58,10 @@ const useSiteMenuItems = () => {
 		return allSitesMenu();
 	}
 
+	if ( isJetpack && ! isAtomic ) {
+		return jetpackMenu( { siteDomain } );
```

Comment:
Can we use this as a fallback so we prioritize the dynamic menu that we improved with https://github.com/Automattic/jetpack/pull/18586?

Reply:
Sure, e650a46